tests/approaches/test_interactive_approach.py

```diff
@@ -27,6 +27,7 @@ def test_interactive_learning_approach():
         "teacher_dataset_label_ratio": 1.0,
         "num_train_tasks": 5,
         "num_test_tasks": 5,
+        "option_learner": "neural",
```

Comment:
what's going on here? can we revert this once you address tom's comment? it might slow down this test